Source/Core/Core/CoreTiming.cpp

```diff
@@ -479,8 +479,8 @@ void Idle()
 
 	idledCycles += DowncountToCycles(PowerPC::ppcState.downcount);
 	PowerPC::ppcState.downcount = 0;
-
-	Advance();
+	if(_M_ARM_32) //Temp Fix
```

Comment:
@lioncash : I know but haven't the time right now and nobody work on ARMv7 except me and Sonicadvance(which concentrate his work on Aarch64). So in three months the problem will still be there.